src/lib.rs

```diff
@@ -33,9 +33,9 @@ pub type BOOLEAN = ::std::os::raw::c_uchar;
 
 /// Family of an IP address.
 pub type AddressFamily = u16;
-pub const ADDRESS_FAMILY_UNSPEC: AddressFamily = c_types::AF_UNSPEC as u16;
-pub const ADDRESS_FAMILY_INET: AddressFamily = c_types::AF_INET as u16;
-pub const ADDRESS_FAMILY_INET6: AddressFamily = c_types::AF_INET6 as u16;
+pub const ADDRESS_FAMILY_UNSPEC: AddressFamily = c_types::AF_UNSPEC;
+pub const ADDRESS_FAMILY_INET: AddressFamily = c_types::AF_INET;
+pub const ADDRESS_FAMILY_INET6: AddressFamily = c_types::AF_INET6;
```

Comment:
Looks like this breaks on other platforms?

Reply:
Fixed by keeping the cast and suppressing the linting.